SlidePuzzle/src/main/java/tests/KeyListenerExample.java

```diff
@@ -0,0 +1,102 @@
+package tests;
+
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.Label;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.util.Arrays;
+public class KeyListenerExample implements KeyListener
```

Comment:
Is this class a test? I was thinking that all the classes in the tests folder are unit tests